packages/graphile-build-pg/src/QueryBuilder.js

```diff
@@ -126,6 +126,7 @@ class QueryBuilder {
   lockContext: {
     queryBuilder: QueryBuilder,
   };
+  children: Map<string, QueryBuilder>;
```

Comment:
Lets rename this to `_children` to make it clear it's not intended to be a public API.

```suggestion
  _children: Map<string, QueryBuilder>;
```